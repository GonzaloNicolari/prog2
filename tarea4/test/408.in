# pruebas para avl_min
avl_min 0
avl_min 1
avl_min 2
avl_min 3
avl_min 4
avl_min 5
avl_min 6	
Fin
