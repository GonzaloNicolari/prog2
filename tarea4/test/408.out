1>#  pruebas para avl_min.
2>
Cantidad: 0.
3>
1 
Cantidad: 1.
4>
1 
2 
Cantidad: 2.
5>
1 
2 4 
3 
Cantidad: 4.
6>
1 
2 4 6 
3 7 
5 
Cantidad: 7.
7>
1 
2 4 6 9 
3 7 10 12 
5 11 
8 
Cantidad: 12.
8>
1 
2 4 6 9 14 
3 7 10 12 15 17 19 
5 11 16 20 
8 18 
13 
Cantidad: 20.
9>Fin.
